modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/tests/RaylifeCreateAccount.testcase

```diff
@@ -194,57 +193,73 @@ definition {
 			locator1 = "RaylifeCreateAccount#SELECT_QUOTE",
 			title = "standard");
 
-		Type(
-			locator1 = "RaylifeCreateAccount#EMAIL_FIELD",
-			value1 = "test@liferay.com");
-
-		Type(
-			label = "password",
-			locator1 = "RaylifeCreateAccount#PASSWORD_FIELD",
-			value1 = "test@123");
-
-		Type(
-			label = "rePassword",
-			locator1 = "RaylifeCreateAccount#PASSWORD_FIELD",
-			value1 = "test@123");
+		Click(
+			key_text = "CREATE NEW ACCOUNT",
+			locator1 = "Button#ANY");
 
-		AssertElementPresent(locator1 = "RaylifeCreateAccount#CREATE_ACCOUNT");
+		AssertElementPresent(locator1 = "RaylifeCreateAccount#CREATE_ACCOUNT_DISABLED");
 	}
 
-	@description = "LPS-144333 - Verify if the create account button is disabled"
-	@priority = 3
-	test IsCreateAccountButtonDisabled {
-		property test.name.skip.portal.instance = "RaylifeCreateAccount#IsCreateAccountButtonDisabled";
+	@description = "LPS-144334 - Verify if after filling in the password and the email with all correct information, the create account button is enabled"
+	@priority = 5
+	test IsCreateAccountButtonEnabled {
+		property solutions.acceptance = "true";
+		property test.name.skip.portal.instance = "RaylifeCreateAccount#IsAccountButtonEnabled";
 
-		ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+		task ("Given that a site is created with the Raylife D2C site template and the flow for getting a quote is started with info inputted") {
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
 
-		RaylifeHomePage.startFlow();
+			RaylifeHomePage.startFlow();
 
-		RaylifeContactInfo.typeAllBasicsInfoFields();
+			RaylifeContactInfo.typeAllBasicsInfoFields();
 
-		Click(locator1 = "Raylife#BUTTON_CONTINUE");
+			Click(locator1 = "Raylife#BUTTON_CONTINUE");
 
-		RaylifeBusiness.typeAllBusinessInfoFields();
+			RaylifeBusiness.typeAllBusinessInfoFields();
 
-		Click(locator1 = "Raylife#BUTTON_CONTINUE");
+			Click(locator1 = "Raylife#BUTTON_CONTINUE");
 
-		RaylifeEmployees.typeAllEmployeesInfoFields();
+			RaylifeEmployees.typeAllEmployeesInfoFields();
 
-		Click(locator1 = "Raylife#BUTTON_CONTINUE");
+			Click(locator1 = "Raylife#BUTTON_CONTINUE");
 
-		RaylifeProperty.typeAllPropertyInfoFields();
+			RaylifeProperty.typeAllPropertyInfoFields();
 
-		Click(locator1 = "Raylife#BUTTON_CONTINUE");
+			Click(locator1 = "Raylife#BUTTON_CONTINUE");
 
-		Click(
-			locator1 = "RaylifeCreateAccount#SELECT_QUOTE",
-			title = "standard");
+			Click(
+				locator1 = "RaylifeCreateAccount#SELECT_QUOTE",
+				title = "standard");
+		}
 
-		Click(
-			key_text = "CREATE NEW ACCOUNT",
-			locator1 = "Button#ANY");
+		task ("When attempting to create a new account with the proper form inputs") {
+			Click(
```

Comment:
Please use the RaylifeCreateAccount.createAccount macro.
This will also change the final assumption since the macro clicks into the account.
We can use:
`task ("Then the user is redirected to the account creation page because the Create Account button is active")`